### DISCUSSION TITLE
PylonCapture Source Node not Populating on new version.

### DISCUSSION THREAD

**Discussion by ab4728** [bonsai-rx/bonsai#1575] [Q&A] [answered]
I can't find the pylon capture node in my toolbox since reinstalling bonsai and updating to the most recent version. Is there a specific package I need to download? I run neurophotometrics through bonsai so I have been just using their packages. Please help.

**Comment by bruno-f-cruz** [comment]
Can you attach your bonsai.config file? it should be located in the same folder as the bonsai.exe executable. thanks!

**Reply by ab4728** [reply]
Hi, thank you for the prompt response. It seems github does not support .config files so I can't attach it. Is there a workaround for this?

**Reply by bruno-f-cruz** [reply]
Rename it to .txt, or other allowed extension format.

**Reply by ab4728** [reply]
[Bonsai.txt](https://github.com/bonsai-rx/bonsai/files/12772560/Bonsai.txt)

**Reply by bruno-f-cruz** [reply]
You dont have any Pylon package installed. Have you tried to install it from the package manager?

**Reply by ab4728** [reply]
![image](https://github.com/bonsai-rx/bonsai/assets/111145910/e86c9d6f-be41-4c1e-8923-29e5fdc802cc)
Should I try downloading all of these packages?

**Reply by bruno-f-cruz** [reply, marked as answer]
Select `Bonsai Packages` as your package source. 
Look for `Bonsai - Pylon Library` and install that package.

**Reply by ab4728** [reply]
Thank you, that did the trick. However, now I'm getting this error, which is the problem I had that led me to reinstalling to begin with. 
![image](https://github.com/bonsai-rx/bonsai/assets/111145910/24bc40b1-1742-403e-adf0-ad1ec2df3cd5)

**Reply by bruno-f-cruz** [reply]
Ok, move this discussion to #1574.
